ui/app/components/sidebar/frame.hbs

```diff
@@ -33,7 +31,7 @@
 
       <:footer>
         {{#if (has-feature "Namespaces")}}
-          <NamespacePicker @namespace={{@namespace}} />
+          <NamespacePicker @namespace={{this.clusterController.namespaceQueryParam}} />
```

Comment:
This wasn't actually an issue in the app since a full refresh happens on namespace change but a test was failing since changing the namespace was not triggering an update in the component. In any case `@namespace` was not an arg on the frame component so it's now being passed in as expected.